docs/BUILD.md

```diff
@@ -0,0 +1,69 @@
+# Building Kubicorn
+
+## Make
```

Comment:
I'm not sure would it be good idea to link to Go install tutorial - https://golang.org/doc/install
There is everything needed to know how to install Go for beginners